apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.IntegrationTests/EntryData.cs

```diff
@@ -27,5 +27,7 @@ public static class EntryData
         /// Returns a formatted message based on the two parametes.
         /// </summary>
         public static string GetMessage(string message, string id) => $"{message} - {id}";
+
+        public static string SpanIdToHex(ulong? spanId) => string.Format("0x{0:X}", spanId);
```

Comment:
This isn't the same as the SpanIdToHex local method above, which returns null if spanId is null. Is this ever actually called with a null spanId?